usr.sbin/rtadvd/config.c

```diff
@@ -1435,6 +1490,19 @@ make_packet(struct rainfo *rai)
 		buf += sizeof(struct nd_opt_mtu);
 	}
 
+	if (rai->rai_pref64.p64_enabled) {
+		ndopt_pref64 = (struct nd_opt_pref64 *)buf;
+		ndopt_pref64->nd_opt_pref64_type = ND_OPT_PREF64;
+		ndopt_pref64->nd_opt_pref64_len = 2;
+		ndopt_pref64->nd_opt_pref64_sl_plc =
+			(htons(rai->rai_pref64.p64_sl << 3))
+			| htons((rai->rai_pref64.p64_plc & 0x7));
```

Comment:
Per style(9) the operator (| in this case) should be at the end of a line.

Reply:
fixed.